projects/1.18/assets/epic-samurais/epicsamurai/lang/zh_cn.json

```
@@ -87,7 +87,7 @@
     "item.epicsamurai.brass_samurai_leggings": "机械武士护腿",
     "item.epicsamurai.brass_samurai_boots": "机械武士靴子",
     "item.epicsamurai.brass_repair_kit": "机械武士盔甲维修构件",
-    "item.epicsamurai.oni_maask": "般若面具",
+    "item.epicsamurai.oni_maask": "般若面具 - 开发中",

```

```suggestion
    "item.epicsamurai.oni_maask": "厉鬼面具 - 开发中",
```
仔细看了一下，`Oni Mask`还真不是般若面具，般若是女鬼。（打脸了
同理，可以考虑将上方的`Oni`也翻译作`厉鬼`，日语里`鬼`这个形象是一种高大严厉的形象。